plugins/by-name/treesitter-textobjects/deprecations.nix

```
@@ -0,0 +1,102 @@
+lib: {
+  imports =
+    let
+      basePluginPath = [
+        "plugins"
+        "treesitter-textobjects"
+      ];
+      settingsPath = basePluginPath ++ [ "settings" ];
+
+      mkChangedKeymapOption =
+        oldSubPath: newSubPath:
+        let
+          oldPath = basePluginPath ++ oldSubPath;
+          newPath = settingsPath ++ newSubPath;
+
+          processKeymaps = lib.mapAttrs (
+            key: mapping:
+            if lib.isString mapping then
+              mapping
+            else
+              lib.warn
+                ''
+                  WARNING: `${lib.showOption newPath}` will not convert the sub-option names from camelCase to snake_case.
```

Seeing as this only applies to `query_group`, perhaps we could be more specific in this warning?


```suggestion
                  WARNING: `${lib.showOption newPath}` will expect `query_group` instead of `queryGroup`.
```

---

Done!